docs/tutorials/quickstart.md

```diff
@@ -74,108 +74,48 @@ Note, we have launched a browser for you to login. For old experience with devic
 You have logged in. Now let us find all the subscriptions to which you have access...
 ```
 
-Next, we'll create a resource group. A resource group is a container that holds related resources for an Azure solution. In Azure, you logically group related resources such as storage accounts, virtual networks, and virtual machines (VMs) to deploy, manage, and maintain them as a single entity. In this case, we want to deploy, manage and maintain the whole Kubernetes cluster as a single entity.
```

Comment:
These steps are actually unnecessary, so are not in the spirit of "quick start" for first time users.

Reply:
👍  I have wondered before why we weren't letting `aks-engine deploy` create the RG for us...